packages/itwinui-react/src/styles.js/styles.module.css

```diff
@@ -3,7 +3,7 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
-@layer itwinui-v1, itwinui.v1, itwinui.v2, itwinui.v3;
+@layer itwinui-v1, itwinui.v1, itwinui.v2, itwinui.v3, itwinui.bridge;
```

Comment:
I've realized just now that we should still include `@layer reset` in here as backup, just in case the consuming application does not do it manually.

We will recommend consumers to set the correct layer order in their `<head>`. The first occurrence of a layer name gets used for the purpose of ordering, so our fallback will just be ignored (so there's no downside in doing os).